### DISCUSSION TITLE
Importing salt-ag-theme bloated JS bundle

### DISCUSSION THREAD

**Discussion by priyassg** [jpmorganchase/salt-ds#3042] [Q&A] [answered]
Hello, 
When I do import '@salt-ds/ag-grid-theme/salt-ag-theme.css' my JS bundle size increase by >4MB.
Is there a way to import a minified version. One solution I found is to use the minified version provided by jsdelivr (https://cdn.jsdelivr.net/npm/@salt-ds/ag-grid-theme@1.3.8/salt-ag-theme.min.css) Wondering if there is any other way I don't know about?

**Comment by joshwooding** [comment, marked as answer]
Right now there's not much you can do apart from looking to see what specifically is causing the increase. The files aren't 4MB so something weird is happening there. But we're looking to improve the size of our theme in the next major release. https://github.com/jpmorganchase/salt-ds/issues/2103
